pandas/tseries/tests/test_base.py

```diff
@@ -45,6 +45,30 @@ def test_ops_properties_basic(self):
         self.assertEqual(s.day,10)
         self.assertRaises(AttributeError, lambda : s.weekday)
 
+    def test_astype_str(self):
+        # test astype string
+        result = date_range('2012-01-01', periods=4, name='test_name').astype(str)
+        expected = Index(['2012-01-01', '2012-01-02', '2012-01-03','2012-01-04'],
+                            name='test_name', dtype=object)
+        tm.assert_index_equal(result, expected)
+
+        # test astype string with tz and name
+        result = date_range('2012-01-01', periods=3, name='test_name', tz='US/Eastern').astype(str)
+        expected = Index(['2012-01-01 00:00:00-05:00', '2012-01-02 00:00:00-05:00', '2012-01-03 00:00:00-05:00'],
+                         name='test_name', tz='US/Eastern', dtype=object)
```

Comment:
the ``Index`` should not have a ``tz`` on the expected (not an error because we allow arbitrary kwargs), but take it out

also pls format this over 2 or more lines (PEP)